lobsterpy/plotting/__init__.py

```diff
@@ -219,4 +219,5 @@ def _broaden(energies: np.ndarray, population: np.ndarray, sigma=None, cutoff=4.
         kernel_x = np.concatenate([-kernel_x[-1:1:-1], kernel_x])
 
         kernel = norm.pdf(kernel_x, scale=sigma)
-        return convolve(population, kernel, mode="same")
+
+        return convolve(population, kernel, mode="same") / sum(kernel)
```

Comment:
That'll do it, cheers!

`kernel.sum()` is much faster than the generic `sum`, but it's microseconds so not really a big deal.

Reply:
Great. Will make this change as well and then use this for plotting. Thanks!